runtime/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/AbstractSchemaRegistryOptionsConfigAdapter.java

```diff
@@ -187,8 +191,23 @@ public OptionsConfig adaptFromJson(
                 JsonObject credentials = object.getJsonObject(AUTHORIZATION_CREDENTIALS_NAME);
 
                 JsonObject headers = credentials.getJsonObject(AUTHORIZATION_CREDENTIALS_HEADERS_NAME);
+                JsonObject basic = credentials.getJsonObject(AUTHORIZATION_CREDENTIALS_BASIC_NAME);
 
-                options.authorization(headers.getString(AUTHORIZATION_NAME));
+                String authorization;
+
+                if (headers != null)
+                {
+                    authorization = headers.getString(AUTHORIZATION_NAME);
+                }
+                else
+                {
+                    String username = basic.getString(AUTHORIZATION_CREDENTIALS_BASIC_USERNAME_NAME);
+                    String password = basic.getString(AUTHORIZATION_CREDENTIALS_BASIC_PASSWORD_NAME);
+                    authorization = "Basic " + Base64.getEncoder().encodeToString(
```

Comment:
we can handle this logic in `AbstractSchemaRegistryOptionsConfigBuilder` instead of adapter.